templates/content/node--publication--search-result.html.twig

```diff
@@ -0,0 +1,13 @@
+{#
+/**
+ * @file
+ * Theme override to display a publication node in search result view mode.
+#}
+<h3><a href="{{ url }}">{{ label }}</a></h3>
+<p class="meta">
+  <span class="label-inline">{{ content.field_published_date | field_label }}</span>
+  <span class="date-display-single">{{ content.field_published_date | field_value }}</span>
+  <span class="metaItemList">{{ content.field_publication_type | field_value }}</span>
```

Comment:
Class name doesn't follow the same case style as the span's above this.

Reply:
You're right. This is taking the present markup as it's found on the D7 site.

@David-Corbett @neilblair - would you like `metaItemList` converted to `meta-item-list` or remain as it is? If the latter, is there any significance to the camelcase style?

Reply:
convert it for now, it should probably be a more meaningful name and possibly might not even be needed. We can review when we come to theming the page